src/Symfony/Bundle/FrameworkBundle/Templating/Helper/CodeHelper.php

```diff
@@ -31,7 +31,7 @@ class CodeHelper extends Helper
     public function __construct($fileLinkFormat, $rootDir, $charset)
     {
         $this->fileLinkFormat = $fileLinkFormat ?: ini_get('xdebug.file_link_format') ?: get_cfg_var('xdebug.file_link_format');
-        $this->rootDir = str_replace('\\', '/', $rootDir).'/';
+        $this->rootDir = str_replace('\\', '/', dirname($rootDir)).'/';
```

Comment:
that's a bug fix, since the root_dir is never a prefix of a PHP source file.
Matches what CodeExtension already does in TwigBundle

Reply:
Or we could pass `kernel.project_dir` when configuring it instead